server/src/test/java/org/eclipse/lsp/cobol/usecases/TestSetUpDownBy.java

```diff
@@ -55,7 +56,7 @@
           + "           SET {$NOT-A-TABLE|1} {$IND1} UP BY 1.\n"
           + "           SET {$IND1} UP BY {$INTVAL}.\n"
           + "           SET {$IND1} UP BY {$SUBINT} OF {$GROUP-VAR}.\n"
-          + "           SET {$IND1} UP BY {$SUBINT} {IND3|3}.\n"
+          + "           SET {$IND1} UP BY {$SUBINT} {#*IND3|3|5}.\n"
```

Comment:
Why do you mark it as a paragraph definition?

Reply:
This is a planned thing.